ffv1.md

```diff
@@ -467,36 +467,37 @@ Third is the end of range coded Slices which need to terminate before the CRC at
 
 ```c
-pseudo-code                                                   | type
---------------------------------------------------------------|-----
-void put_symbol(RangeCoder *c, uint8_t *state, int v, int \   |
-is_signed) {                                                  |
-    int i;                                                    |
-    put_rac(c, state+0, !v);                                  |
-    if (v) {                                                  |
-        int a= abs(v);                                        |
-        int e= log2(a);                                       |
-                                                              |
-        for (i = 0; i < e; i++) {                             |
-            put_rac(c, state+1+min(i,9), 1);  //1..10         |
-        }                                                     |
-                                                              |
-        put_rac(c, state+1+min(i,9), 0);                      |
-        for (i = e-1; i >= 0; i--) {                          |
-            put_rac(c, state+22+min(i,9), (a>>i)&1); //22..31 |
-        }                                                     |
-                                                              |
-        if (is_signed) {                                      |
-            put_rac(c, state+11 + min(e, 10), v < 0); //11..21|
-        }                                                     |
-    }                                                         |
-}                                                             |
+int get_symbol(RangeCoder *c, uint8_t *state, int is_signed) {
+    if (get_rac(c, state + 0) {
+        return 0;
+    }
+
+    int e = 0;
+    while (get_rac(c, state + 1 + min(e, 9)) { //1..10
+        e++;
+    }
+
+    int a = 1;
+    for (int i = e - 1; i >= 0; i--) {
+        a = a * 2 + get_rac(c, state + 22 + min(i, 9));  // 22..31
+    }
+
+    if (!is_signed) {
+        return a;
+    }
+
+    if (get_rac(c, state + 11 + min(e, 10))) { //11..21
+        return -a;
+    } else {
+        return a;
+    }
+}
 ```
 Figure: A pseudo-code description of the contexts of Range Non Binary Values. {#figureRangeNonBinaryValueExample}
 
-`put_symbol` is used during the process of (#coding-of-the-sample-difference).
+`get_symbol` is used during the process of (#coding-of-the-sample-difference).
 
-`put_rac` is the process described in (#range-binary-values).
+`get_rac` is the process described in (#range-binary-values).
```

Comment:
This is rendered as 
> "get_rac" is the process described in Section 3.8.1.1.

But Section 3.8.1.1 refers to several processes. Could we add a "Figure" label to the process you mean and cross-reference that rather than the whole section. For instance, see how `{#figureRelativeSampleNames}` is cited.

Reply:
> But Section 3.8.1.1 refers to several processes.

IMO `get_rac` is for all figures in the section, which describe a single process, so the link is correct. I don't see a better way to link, @michaelni would you have a suggestion here about how to improve this description?

***

I think that no modification is necessary here, and all other issues are fixed, please review again.